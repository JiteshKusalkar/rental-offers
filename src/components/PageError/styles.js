import styled from 'styled-components';

const Wrapper = styled.div`
  height: 100vh;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
`;
const ErrorCode = styled.p`
  font-size: 200px;
  color: #40bca7;
  margin: 0;
`;
const Description = styled.span``;

export { Description, ErrorCode, Wrapper };
