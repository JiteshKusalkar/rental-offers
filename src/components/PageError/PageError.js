import { Description, ErrorCode, Wrapper } from './styles';

function PageError() {
  return (
    <Wrapper>
      <ErrorCode>404</ErrorCode>
      <Description>Page could not be loaded</Description>
    </Wrapper>
  );
}

export default PageError;
