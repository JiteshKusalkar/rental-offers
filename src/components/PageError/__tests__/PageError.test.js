import { render } from '@testing-library/react';
import PageError from '../PageError';

describe('PageError', () => {
  it('should render the component', () => {
    const { container } = render(<PageError />);

    expect(container).toMatchSnapshot();
  });
});
