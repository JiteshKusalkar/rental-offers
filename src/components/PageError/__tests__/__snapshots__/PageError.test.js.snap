// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageError should render the component 1`] = `
<div>
  <div
    class="sc-bdvvtL tuHyk"
  >
    <p
      class="sc-gsDKAQ gIbonr"
    >
      404
    </p>
    <span
      class="sc-dkPtRN"
    >
      Page could not be loaded
    </span>
  </div>
</div>
`;
